src/main/java/athena/Athena.java

```diff
@@ -13,24 +14,23 @@ public Athena() {
     public static void main(String[] args) {
         Athena athena = new Athena();
         athena.runProgram();
-        Storage storage = new Storage("save.csv");
-        TaskList oldTask = new TaskList();
-        oldTask = storage.loadTaskListData();
-
-
-
+        Storage storage = new Storage("save.csv",ui);
         TaskList taskList = new TaskList();
+        taskList = storage.loadTaskListData();
         Parser parser = new Parser();
         Command command = parser.parse("add n/Assignment2 t/1100 d/16-09 D/2 r/Monday i/high a/Refer to lecture notes");
         command.execute(taskList, athena.ui);
-        command = parser.parse("add n/Assignment2 t/1100 d/16-09 D/2 r/Monday i/high a/Refer to lecture notes");
-        command.execute(taskList, athena.ui);
+        for (int i = 0; i < 10; i++) {
+            command = parser.parse("add n/Assignment2 t/1100 d/16-09 D/2 r/Monday i/high a/Refer to lecture notes");
+            command.execute(taskList, athena.ui);
+        }
```

Comment:
Would be nice to make this a test case.

Reply:
I'll just keep it in for now until Alston is done with the main Athena